<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISL_FIS_KOPYALAMA_BATCH</class>
 <widget class="QDialog" name="ISL_FIS_KOPYALAMA_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>İŞLETME DEFTERİ FİŞ KOPYALAMA</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1" columnstretch="350,2">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>484</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_batch_buttons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1" columnstretch="5,5,2,5,5">
     <item row="1" column="0">
      <widget class="QLabel" name="label_fis_no">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fiş No </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSearchEdit" name="search_edit_fis_no">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Arama widget. </string>
       </property>
       <property name="QSearchEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_aciklama">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Açıklama </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QTextEdit" name="textedit_aciklama">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_yeni_fis_no">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Yeni Fiş No </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_yeni_fis_no">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_yeni_fis_tarihi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Yeni Fiş Tarihi</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_yeni_aciklama">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Kopyalama işlemi sonucunda oluşturulacak olan yeni fişin açıklamasını gösterir.</string>
       </property>
       <property name="text">
        <string>Yeni Açıklama</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="4">
      <widget class="QTextEdit" name="textedit_yeni_aciklama">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_fis_turu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fiş Türü</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_fis_turu">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fiş Tarihi</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QAdakDate" name="adakDate_kpylnck_fis_tarihi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>242</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>242</width>
         <height>30</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Tarih WidgetÄ± </string>
       </property>
       <property name="QAdakDate" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QAdakDate" name="adakDate_yeni_fis_tarihi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>242</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>242</width>
         <height>30</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Tarih WidgetÄ± </string>
       </property>
       <property name="QAdakDate" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAdakDate</class>
   <extends>QWidget</extends>
   <header>QAdakDate.h</header>
  </customwidget>
  <customwidget>
   <class>QSearchEdit</class>
   <extends>QWidget</extends>
   <header>QSearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>adakDate_kpylnck_fis_tarihi</tabstop>
  <tabstop>search_edit_fis_no</tabstop>
  <tabstop>lineEdit_yeni_fis_no</tabstop>
  <tabstop>adakDate_yeni_fis_tarihi</tabstop>
  <tabstop>textedit_yeni_aciklama</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
